config/oidc_backends_config.xml.sample

```diff
@@ -22,4 +22,33 @@
          login to Galaxy using their Google account.
         -->
     </provider>
+    <provider name="Google" implementation="OIDC">
```

Comment:
`OIDC` refers to all the providers, so I guess it would be better to name this `implementation="keyclock"` instead.